practice_app/pages/api/getStatistics.js

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios'
+
+const { checking } = require('./getStatisticsModule')
+
+export default (req, res) => {
+    // req object contains whatever information we can get from the request that comes from the client (from the browser)
+    // res object is used tow write our reponse
+    const { query, method } = req
+
+    switch (method) {
+        case 'GET':
+            {
+                // ...
+                res.statusCode = 200
+
+                const countryName = query.country
+
+                let deathToll1
+                let recovered1
+                let answered1
+                let infected1
+
+                axios
+                    .get(
+                        'https://api.apify.com/v2/key-value-stores/tVaYRsPHLjNdNBu7S/records/LATEST?disableRedirect=true',
+                    )
+
+                    .then(response => {
+                        let i
+
+                        for (i = 0; i < response.data.length; i++) {
+                            if (response.data[i].country == countryName) {
+                                answered1 = 'yes'
+                                deathToll1 = response.data[i].deceased
+                                recovered1 = response.data[i].recovered
+                                infected1 = response.data[i].infected
+
+                                break
+                            }
+                        }
```

Comment:
I suggest you use `Array.find` with an anonymous function to find the country which matches `countryName`. No need for `for` loops and break and stuff.

So you can `find` then take out `deceased, recovered, infected` out of it.

You can read about it here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Thats a good suggestion but in terms of complexity i suppose they are the same, if so i would rather not change the code but next time i would use it for sure.

Reply:
The complexity is indeed the same. Using `map`, `find`, `reduce` etc is often more readable and more idiomatic Javascript. Next time keep it in mind ;)